Amplify/DevMenu/DevEnvironmentInfo.swift

```diff
@@ -9,12 +9,12 @@ import Foundation
 
 // struct to decode/encode information about developer environment in json format
 struct DevEnvironmentInfo: Codable {
-    let nodejsVersion: String
-    let npmVersion: String
-    let amplifyCLIVersion: String
-    let podVersion: String
-    let xcodeVersion: String
-    let osVersion: String
+    let nodejsVersion: String?
+    let npmVersion: String?
+    let amplifyCLIVersion: String?
+    let podVersion: String?
+    let xcodeVersion: String?
+    let osVersion: String?
```

Comment:
This will show `Not Available` if the value is nil?

Reply:
Yep, that is the way the Dev Menu view is currently written.